packages/react-components/src/trace-explorer/trace-explorer-views-widget.tsx

```diff
@@ -42,11 +42,13 @@ export class ReactAvailableViewsWidget extends React.Component<ReactAvailableVie
         });
         signalManager().on(Signals.EXPERIMENT_SELECTED, this.doHandleExperimentSelectedSignal);
         signalManager().on(Signals.CHECK_OPEN_WIDGETS, this.doHandleCheckOpenWidgetsSignal);
-        this.state = { availableOutputDescriptors: [] };
+        this.state = { availableOutputDescriptors: [], lastSelectedOutputIndex: -1 };
```

Comment:
I don't think this commit from master branch should be merged in this branch, instead this branch should be rebased on the latest master branch commit.